### PR TITLE
Translatable remove calendar message and else styling

### PR DESCRIPTION
**Pull request by vgezer**
@georgehrke please review. thanks.

### CODE REVIEW

File: js/calendar.js [hunk 1]
```diff
@@ -124,7 +124,8 @@ Calendar={
 		loading: function(isLoading){
 			if (isLoading){
 				$('#loading').show();
-			}else{
+			}
+			else {
```
Comment (georgehrke) [inline js/calendar.js]: please revert this change

Comment (georgehrke) [inline js/calendar.js]: as well as all the other } else to } \n else

Comment (vgezer) [inline js/calendar.js]: This makes the code search harder since most of the code uses this styling. Even in this file...?

Comment (vgezer) [inline js/calendar.js]: Just one example: https://github.com/owncloud/calendar/blob/minor/js/calendar.js#L128

Comment (georgehrke) [inline js/calendar.js]: http://doc.owncloud.org/server/6.0/developer_manual/app/general/codingguidelines.html#id2

Comment (vgezer) [inline js/calendar.js]: So do you think better i replace `}else{` with `} else {` for the same reason, or ignore them completely?

Comment (georgehrke) [inline js/calendar.js]: I'd say ignore them completely

**Comment by georgehrke** [comment]
thx :)
